source/auth/auth.rst

```diff
@@ -179,6 +179,18 @@ specified and has not been negotiated for that credential:
   instructions for when the ``saslSupportedMechs`` field is not present in
   an ``isMaster`` response.
 
+Caching credentials in SCRAM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the implementation of SCRAM authentication mechanisms (e.g. SCRAM-SHA-1
+and SCRAM-SHA-256), drivers MUST maintain a cache of computed SCRAM credentials.
+The cache entries SHOULD be identified by some combination of the password, 
```

Comment:
What does "some combination" mean?  Can we be more explicit?

Reply:
I originally had written it to be described as map with tuples, but @xdg commented earlier:

"I don't like that this is getting into specific mechanism like keys/tuples -- we should focus on "what", not "how". The important thing is to be clear what factors uniquely determine a cached value. How a library establishes uniqueness is irrelevant."

Reply:
Would removing "some combination of" violate the spirit of that comment?

Reply:
That seems reasonable to me; I'll make the change.